x-pack/plugins/cloud_security_posture/public/pages/benchmarks/benchmarks_table.tsx

```diff
@@ -30,46 +30,56 @@ interface BenchmarksTableProps
 }
 
 const AgentPolicyButtonLink = ({ name, id: policyId }: { name: string; id: string }) => {
-  const { http } = useKibana().services;
+  const { http, application } = useKibana().services;
   return (
-    <EuiLink
+    <a
```

Comment:
Why not use `EuiLink`?

Reply:
it can't take both `href` and `onClick` (custom eslint error) 

and i need both:
- `href` so the link is shareable
 - `onClick` so we can navgiate to external react-app (fleet) or let it propagate to `RedirectAppLinks` (but we can't, since we don't want to propagate to the row click)